Assets/_BForBoss/_Weapons/Scripts/ScriptableObjects/Bullets/RayCastBulletPropertiesScriptableObject.cs

```diff
@@ -0,0 +1,18 @@
+using UnityEngine;
+
+namespace Perigon.Weapons
+{
+    
+    [CreateAssetMenu(fileName = "BulletProperties", menuName = "PerigonGames/RayCastBullet", order = 2)]
+    public class RayCastBulletPropertiesScriptableObject : ScriptableObject, IBulletProperties
+    {
+        [SerializeField] private float _damage = 1f;
+        [SerializeField] private float _maxDistance = 100;
+        [SerializeField] private float _bulletHoleTimeToLive = 10f;
+
+        public float Damage => _damage;
+        public float Speed => 0;
```

Comment:
What is the point of this property?

Reply:
Because it's part of the `IBulletProperties` I had to fill these in.